exchanges/deribit/deribit.go

```diff
@@ -357,20 +357,18 @@ func (d *Deribit) GetInstrument(ctx context.Context, instrument string) (*Instru
 
 // GetInstruments gets data for all available instruments
 func (d *Deribit) GetInstruments(ctx context.Context, ccy currency.Code, kind string, expired bool) ([]InstrumentData, error) {
```

Comment:
Suggesting we make this pointers.
It has a few knock-ons, which are in [this git-diff want to accept it](https://gist.github.com/gbjk/3a54e583e9fd96e93f13745220b822b2).
```suggestion
func (d *Deribit) GetInstruments(ctx context.Context, ccy currency.Code, kind string, expired bool) ([]*InstrumentData, error) {
```